apps/gallery/js/thumbnail_item.js

```diff
@@ -39,7 +39,9 @@ ThumbnailItem.formatter = new navigator.mozL10n.DateTimeFormat();
 
 ThumbnailItem.prototype.localize = function() {
   var date = new Date(this.data.date);
-  var imageCreatedAt = navigator.mozL10n.get('image-created-at');
-  var label = ThumbnailItem.formatter.localeFormat(date, imageCreatedAt);
+  var description = navigator.mozL10n.get(
+    !!this.data.metadata.video ?
+    'imageDescriptionShort' : 'videoDescriptionShort');
```

Comment:
This should be other way around i.e !!this.data.metadata.video ?
+ 'videoDescriptionShort' : 'imageDescriptionShort'